sapi/fpm/tests/gh11066-fpm-unix-socket-too-long-filename-but-starts-anyway.phpt

```diff
@@ -0,0 +1,66 @@
+--TEST--
+FPM: gh11066 - listen filename is too long for OS socket path limitation but starts anyway
+--SKIPIF--
+<?php
+include "skipif.inc"; ?>
+--FILE--
+<?php
+
+require_once "tester.inc";
+$tempDir = sys_get_temp_dir();
```

Comment:
it would be better to do it in the test dir and use test naming as it's the case for other tests - it could maybe use some helper function for tester (that's not necessary if you find it too hard but might be nicer).

Reply:
I am not sure where a helper method would simplify this so I solved it as suggested